java/src/main/native/src/ColumnVectorJni.cpp

```diff
@@ -38,12 +38,70 @@
 #include <rmm/mr/device/per_device_resource.hpp>
 
 #include <arrow/api.h>
+#include <arrow/c/bridge.h>
 
 #include <algorithm>
 
 using cudf::jni::ptr_as_jlong;
 using cudf::jni::release_as_jlong;
 
+// Creating arrow as per given type_id and buffer arguments
```

Comment:
Yup certainly, where would you like this header to live?

Reply:
Wait sorry, I think there's some confusion here. This code was only pasted into one Java file. The other files is a C++ test file. My expectation is that the latter will persist indefinitely because we want to continue testing this interop, whereas the Java copy in this file should be temporary until the JNI is updated to no longer use libarrow APIs anymore. That's why I chose to leave two distinct copies instead of creating a shared header.

Reply:
never mind then